plugins/inputs/logstash/README.md

```diff
@@ -0,0 +1,83 @@
+# Logstash Input Plugin
+
+This plugin reads metrics exposed by Logstash Monitoring API.
```

Comment:
Can you link to the documentation for this API here?  Maybe https://www.elastic.co/guide/en/logstash/current/monitoring-logstash.html